docs/dev-guide/integration/rest-api/consumer/applicability.rst

```diff
@@ -87,6 +85,13 @@ and postponed until the current task is completed.
   }
  }
 
+:group_call_report:`_` ::
+
+ {
+     "_href": "/pulp/api/v2/task_groups/16412fcb-06fa-4caa-818b-b103e2a9bf44/state_summary/",
```

Comment:
FWIW I'm not crazy about this, but I'm not sure we can do better right now. In theory the ``_href`` should reference the root of the resource, but I recognize that we probably haven't implemented that endpoint yet (or have we?). In an ideal world, our REST API would have a section for links, one of which could be ``_href``, among others. Like this:

```
{
  "_links": {
    "href": "/pulp/api/v2/task_groups/16412fcb-06fa-4caa-818b-b103e2a9bf44/",
    "state_summary": "/pulp/api/v2/task_groups/16412fcb-06fa-4caa-818b-b103e2a9bf44/state_summary/",
  }
}

In which case we could actually just leave out the ``href`` for now. But we don't have that structure, so we probably can't.

On balance, I think it's better to do the most correct option, have the ``_href`` reference the root of the resource, and make the API client add ``state_summary`` if they want it.